### DISCUSSION TITLE
Welcome to bench-warmers Discussions!

### DISCUSSION THREAD

**Discussion by dmarx** [dmarx/bench-warmers#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by un1tz3r0** [comment]
Hi, my name is Victor Condino, aka @un1tz3r0 on twitter/linktree/objkt/various discords etc. I am a self-taught programmer, musician, artist and thinker with a passion for machine learning/AI and making the world a better, safer and more fair and inclusive place with art and technology.

I believe that it is the responsibility of those who have great ideas to share them with the world, so I like this repo a lot... I'm wondering whether I should start my own in the same spirit or submit some PRs with my own potential projects?

Just scanning through the list I see a couple things I thought of too. Thanks for putting this together, these are all great and all together really inspirational.

I will try and go back over the list in more detail to see if there's anything that is close to something I have in the works. I'm terrible about cleaning up, documenting and publishing my code, something I have been working hard on improving lately.

**Reply by dmarx** [reply]
great questions!

1. I strongly encourage you to start your own version of this to collect and share your own ideas.
  - I've been meaning to turn this into a template to make it easier for other people to do exactly that, but I want to improve a few things here first.
  - For now: https://github.com/dmarx/bench-warmers/blob/main/FAQ.md#id-sorta-like-to-make-my-own-version-of-this-how-would-i-do-that
2. If you would like to contribute to ideas that are already described in my workbench, feel free to submit a PR to add your notes

**Reply by dmarx** [reply]
Hey Victor! I finally got around to turning it into a template repository with setup instructions! take it for a spin, lemme know what you think: https://github.com/dmarx/workbench
